src/containers/galaxy-selector-container.scss

```diff
@@ -3,36 +3,43 @@
     padding: 9px;
```

Comment:
Please run the style linter on your new style files and fix and warnings or errors that pop up

Reply:
Felt weird to duplicate lines in css in order to abide by this "no more that 5 nesting depth" rule, but it is fixed.